src/command_modules/azure-cli-acs/azure/cli/command_modules/acs/custom.py

```diff
@@ -509,6 +509,9 @@ def acs_create(resource_group_name, deployment_name, name, ssh_key_value, dns_na
     if location is None:
         location = rg.location  # pylint:disable=no-member
 
+    # normalize location, such as "West US" -> "westus"
+    location = location.replace(" ", "").lower()
```

Comment:
Good catch! 

`location` already does the normalization trick. The PR is not needed.